Plugins/agsblend/AGSBlend.cpp

```diff
@@ -42,18 +42,16 @@ typedef unsigned char uint8;
 #define DEFAULT_RGB_B_SHIFT_32  0
 #define DEFAULT_RGB_A_SHIFT_32  24
 
-#if !AGS_PLATFORM_OS_WINDOWS
-#define min(x,y) (((x) < (y)) ? (x) : (y))
-#define max(x,y) (((x) > (y)) ? (x) : (y))
-#endif
+#define math_min(x,y) (((x) < (y)) ? (x) : (y))
+#define math_max(x,y) (((x) > (y)) ? (x) : (y))
```

Comment:
It's better to go opposite way and disable macros coming from windows.h. Not only having to rename macros annoying, but also windows.h macros will prevent using std::min/max if one wishes to switch to these.

First I'd double check why windows.h is included here, but likely it's because of allegro.h
Then there are two ways the window header may be dealt with:
1. Undef-ing conflicting macros after the header.
2. Defining `#define NOMINMAX` before the header - in which case windows.h will skip these macro definition.

For example see 
https://github.com/adventuregamestudio/ags/blob/master/Common/platform/windows/windows.h

Reply:
I noticed a few things were actually supposed to be built only in non-builtin plugins, so I adjusted that, but also used the NOMINMAX define there and used std::min and std::max instead.